README.md

```diff
@@ -139,27 +138,63 @@ $ git push
 #### 22. What is the difference between `git add *; git commit` and `git commit -a -m`?
 #### 23. What is the difference between `pull` and `fetch`?
 #### 24. How to make a commit from another author?
+
+```bash
+$ git commit --author "Name Surname <email@address>" ...
+```
+
 #### 25. How to create a branch?
 #### 26. How to switch to a branch?
 #### 27. How to know when a file was first added to the repository?
 #### 28. How to get a list of all deleted files?
 #### 29. How to completely remove (sensitive) files from history?
 #### 30. How to list remote repositories?
+
+```bash
+$ git remove -v
+```
+
 #### 31. How to tag a specific commit?
-#### 32. How to get a list of tag?
+
+```bash
+$ git tag -m '...' <commit>
+```
+
+#### 32. How to get a list of tags?
+
+```bash
+$ git tag
+```
+
 #### 33. How to merge 3 last commits into a single one?
 #### 34. How to get last modification for a given file?
 #### 35. How to swap branch master with another one?
 #### 36. How to access all commits in case of problem?
 #### 37. How to get all the commits from a specific user?
 #### 38. How to ignore the .gitignore file?
 #### 39. How to get the list of unpushed commits?
+
+```bash
+$ git log @{u}..
+```
+
 #### 40. How to compare a file across two different branches?
+
+```bash
+$ git diff <branch-a> <branch-b> -- <file>
+```
+
 #### 41. How to merge two git repositories?
 #### 42. What is the difference between HEAD and ORIG_HEAD?
 #### 43. How to use a global .gitignore?
+
+Create `$XDG_CONFIG_HOME/git/ignore` with a list of patterns.
```

Comment:
In 99.9% of cases its ~/.config, but in principle it could be changed to something else. But it can also be undefined, in which case applications are supposed to fall back to ~/.config. So what I wrote is wrong, in the sense that it does not handle the variable being undefined.

This should be either:
```sh
${XDG_CONFIG_HOME:-$HOME/.config}
```
or maybe just
```sh
~/.config
```
What do you think?

Reply:
I prefer the `~/.config` because I think this notation is more common (but I could be wrong). You can also add an alternate version in your answer, such we don't need to choose between the two.

Reply:
And this variable seems to be empty on OSX.

Reply:
In the end I think using just `~/.config` is better. I updated to do just that.